### DISCUSSION TITLE
Purge seems don't work

### DISCUSSION THREAD

**Discussion by antangible** [tailwindlabs/tailwindcss#5642] [Help]
Hi I'm new to Tailwind, I started the configuration and all seems going well but the purge.

I use npm scripts to execute tasks and in a certain point I have the task for building the CSS like this

```
"buildcss": "NODE_ENV=production npx tailwindcss -i ./public/css/style.css -o ./public/css/style-purged.css",
```

that I run with `npm run buildcss`.

Now, when I open `style-purged.css` it shows everything, I expected it showed only the utility classes that I actually used into HTML markup. 

Furthermore it starts and ends with `/*purgecss start ignore*/` and at the end `/*purgecss end ignore*/` in a way like this

```
/*purgecss start ignore*/
.
.
.
/*purgecss end ignore*/
```

What am I wrong?
Thanks in advice!

**Comment by adamwathan** [comment]
Hey! Can you share a project that demonstrates this issue? Hard to know what's going on without seeing your HTML, CSS, Tailwind config, etc.
